### PR TITLE
Get rid of npm version requirement, enforce via git commit hooks

### PR DESCRIPTION
**Pull request by danecreekphotography**
# Fixes 91

## Description of changes

- Get rid of the requirement to update the version on every branch merge. That was so silly.
- Add requirement to comment in the **Unreleased** section of `CHANGELOG.md` for future manual releases
- Add pre-commit hooks for eslint, prettier, and markdown-lint, which is what `npm version` was really trying to do anyway
- Remove unneeded `npm version` pre- and post-script actions from package.json
- Update the pull request template

## Checklist

n/a, not user-facing changes

### CODE REVIEW

File: src/Trigger.ts [hunk 1]
```diff
@@ -194,7 +194,7 @@ export default class Trigger {
    * @param confidence The confidence level
    * @returns True if the confidence level is with the range that activates the trigger.
    */
-  private confidenceMeetsThreshold(fileName: string, confidence: number): boolean {
+  private confidenceMeetsThreshold(fileName: string, confidence: number) {
```
Comment (danecreekphotography) [inline src/Trigger.ts:197]:
```suggestion
  private confidenceMeetsThreshold(fileName: string, confidence: number): boolean {
```

**Comment by danecreekphotography** [comment]
@mattbho since you had good fun dealing with the nonsense version requirement how do these changes look?

**Comment by mattbho** [comment]
> @mattbho since you had good fun dealing with the nonsense version requirement how do these changes look?

Oh man, husky is a super cool package. I like this!

**Comment by danecreekphotography** [comment]
> Learning something new everyday

Me too 😁

**Pull request merged**